bootloader/src/old_machine_bootloader_windows.h

```diff
@@ -0,0 +1,11 @@
+#ifdef _WIN32
+#include <windows.h>
+#include <wininet.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#pragma comment( lib, "wininet" )
+#pragma comment (lib, "Wininet.lib")
+
+int ping_island(int argc, char * argv[]);
+#endif
```

Comment:
All this should be part of the .C file.
There should only be a single header file that only defines
int ping_island(int argc, char * argv[]);
And we should comment why we keep the same signature as main(), to simplify maintaining the code